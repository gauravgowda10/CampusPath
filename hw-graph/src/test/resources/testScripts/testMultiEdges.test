# Tests adding two nodes and six edges to graph

# Create a graph
CreateGraph multiEdgesGraph

# Add two nodes
AddNode multiEdgesGraph x
AddNode multiEdgesGraph y

# Add six edges
AddEdge multiEdgesGraph x x sameXX
AddEdge multiEdgesGraph y y sameYY
AddEdge multiEdgesGraph x y diffXY1
AddEdge multiEdgesGraph x y diffXY2
AddEdge multiEdgesGraph y x diffYX1
AddEdge multiEdgesGraph y x diffYX2

# Print the nodes and edges in the graph
ListNodes multiEdgesGraph
ListChildren multiEdgesGraph x
ListChildren multiEdgesGraph y
