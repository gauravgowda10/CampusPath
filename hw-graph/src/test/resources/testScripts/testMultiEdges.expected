# Tests adding two nodes and six edges to graph

# Create a graph
created graph multiEdgesGraph

# Add two nodes
added node x to multiEdgesGraph
added node y to multiEdgesGraph

# Add six edges
added edge sameXX from x to x in multiEdgesGraph
added edge sameYY from y to y in multiEdgesGraph
added edge diffXY1 from x to y in multiEdgesGraph
added edge diffXY2 from x to y in multiEdgesGraph
added edge diffYX1 from y to x in multiEdgesGraph
added edge diffYX2 from y to x in multiEdgesGraph

# Print the nodes and edges in the graph
multiEdgesGraph contains: x y
the children of x in multiEdgesGraph are: x(sameXX) y(diffXY1) y(diffXY2)
the children of y in multiEdgesGraph are: x(diffYX1) x(diffYX2) y(sameYY)
